azure-pipelines/templates/ci-git-rebase.yml

```diff
@@ -6,5 +6,5 @@ steps:
     git config user.email 'info@openttd.org'
     git config user.name 'OpenTTD CI'
     git rebase origin/${SYSTEM_PULLREQUEST_TARGETBRANCH}
-  displayName: 'Rebase to $(System.PullRequest.TargetBranch)'
+  displayName: 'Rebase to target branch'
```

Comment:
guess this doesn't work? would be nice to have...

Reply:
Indeed that doesn't work :D Goldplating ;)